mmdeploy/codebase/base/backend_model.py

```diff
@@ -26,6 +27,9 @@ def __init__(self,
         if deploy_cfg is not None:
             ir_config = get_ir_config(deploy_cfg)
             output_names = ir_config.get('output_names', None)
+            if get_partition_config(deploy_cfg) is not None:
+                output_names = get_partition_config(
+                    deploy_cfg)['partition_cfg'][0]['output_names']
```

Comment:
what if the model will be partition into two part?

Reply:
`output_names` should be passed into the class instead of `None`.

Reply:
two stage model will be partitioned into two part, which output_names will you accept?